new/reclassify_amazon_linux_platform_family.md

```diff
@@ -0,0 +1,30 @@
+---
+RFC: unassigned
+Author: Sean OMeara <sean@chef.io>
+Status: Draft
+Type: Standards
+---
+
+# Title
+
+Reclassify Amazon Linux platform_family
+
+## Motivation
+
+   As time passes, Amazon Linux looks less and less like a RHEL clone.
+   Release version numbers do not match EL5, EL6, EL7.
+   Amazon is now a rolling release
+   Different package sets are available
+   Package names and contents are different
+   Service providers are different
+
+## Specification
+
+Change Amazon Linux platform_family from 'rhel' to 'amazon'
```

Comment:
So again this sounds like less of a difference to me than ubuntu-vs-debian, both of which are platform_family of debian.   The package repos and package versions along with package availability are completely different.  There has been at least one totally different behavior in the core service resource:  upstart.  For a lot of case statements though a platform_family of 'debian' that covers the latest thing that ubuntu and debian both do the same (as opposed to fedora-based, suse-based, solaris/aix/bsd, etc.) will work fine.